pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -956,6 +956,10 @@ protected void handleSubscribe(final CommandSubscribe subscribe) {
               ? new KeySharedMeta().copyFrom(subscribe.getKeySharedMeta())
               : emptyKeySharedMeta;
 
+
+        log.debug("Topic name = {}, subscription name = {}, schema is {}", topicName, subscriptionName,
```

Comment:
When debug level, add 'log.isDebugEnabled()'

Reply:
I think it depends. If there are some params like `JSONUtil.toJsonStr(obj)`, `log.isDebugEnabled()` is required, but there are only original Strings.